test/language/arguments-object/10.6-13-a-1.js

```diff
@@ -19,23 +20,11 @@ flags: [noStrict]
 
             var verifyValue = false;
             verifyValue = typeof argObj.callee === "function";
-            
-            var verifyWritable = false;
-            argObj.callee = 1001;
-            verifyWritable = (argObj.callee === 1001);
-
-            var verifyEnumerable = false;
-            for (var p in argObj) {
-                if (p === "callee" && argObj.hasOwnProperty("callee")) {
-                    verifyEnumerable = true;
-                }
-            }
-
-            var verifyConfigurable = false;
-            delete argObj.callee;
-            verifyConfigurable = argObj.hasOwnProperty("callee");
 
 assert(verifyValue, 'verifyValue !== true');
```

Comment:
Use `value: true`

Reply:
`verifyValue` is the result of string comparison of the `typeof` expression, so it's not possible to check the value using `verifyProperty`.

I'll just replace the `verifyValue` assertion with:
```js
assert.sameValue(typeof argObj.callee, "function");
```

Reply:
Oh right, I confused myself. Thanks

Reply:
Fixed.